### ISSUE TITLE
Targeted Multi-User tagging

### ISSUE THREAD

**Issue by AliCihan** [julianlam/nodebb-plugin-mentions#244]
I suggest the usage of @everyone and @here to notify all users in the forum and on the topic respectively... 

please see this:
https://community.nodebb.org/topic/17283/targeted-multi-user-tagging

**Comment by julianlam** [comment]
Seems like a fairly straightforward addition, although I am wary of unintentional (or intentional?) spamming via this mechanism.

**Comment by barisusakli** [comment]
Yeah @everyone is probably overkill, and would blow up the forum when you have a couple million users 😆

**Comment by AliCihan** [comment]
> Seems like a fairly straightforward addition, although I am wary of unintentional (or intentional?) spamming via this mechanism.

Yes, it should only be available for privileged users. Only admins, global moderators, and category moderators are making these important announcements on the topics anyway...  Because I see this easily being used for spamming :D

**Comment by AliCihan** [comment]
looking forward to seeing this in action 😃
